cfgov/unprocessed/apps/filing-instruction-guide/js/fig-sidenav.js

```diff
@@ -22,6 +23,36 @@ let lastTargetIndex;
 
 document.querySelector('.o-footer').classList.add( 'report-global-footer' );
 
+let mobileTOC = buildMobileTOC();
+
+function scrunchIfNeeded() {
+  let screenX = window.innerWidth || document.documentElement.clientWidth || document.getElementsByTagName('body')[0].clientWidth;
+  if (screenX < 900) {
+      sidenav.classList.add( 'scrunch' );
+      sidenav.appendChild(mobileTOC);
+      let tocContent = document.querySelector( '.toc-expandable_content' );
```

Comment:
It looks like you're querying for the class `toc-expandable_content` before it exists, here's the error when I load the page:

<img width="1510" alt="image" src="https://user-images.githubusercontent.com/1060248/171719831-e61c728b-69f8-442f-ba92-33c15cf95b95.png">

The error isn't thrown when the page is loaded on a mobile device because the `toc-expandable_content` class *does* exist for screens with a width less than 900px.

<img width="1510" alt="image" src="https://user-images.githubusercontent.com/1060248/171719684-67fd9433-f200-47c5-a4c7-f5a3f4be1986.png">

So you'll need to add some sort of conditional or error handling to only initialize the expandable on smaller screens.